packages/react/polymorphic/package.json

```diff
@@ -18,6 +18,7 @@
     "prepublish": "yarn clean && yarn build"
   },
   "devDependencies": {
+    "@radix-ui/react-primitive": "workspace:*",
```

Comment:
we don't normally put dev deps (of our packages) in here otherwise it screws up with yarn's versioning, ie, when changing primitives, it'll ask for release of polymorphic, which is wrong.